test/system/090-events.bats

```diff
@@ -33,11 +33,18 @@ load helpers.network
     run_podman events --since "$before" --filter container=$cname --filter event=start --stream=false --no-trunc=false
     is "$output" ".* $truncID " "filtering by container name trunc id"
 
+    # Wait for container to truly be gone.
+    # 99% of the time this will return immediately with a "no such container" error,
+    # which is fine. Under heavy load, it might actually catch the container while
+    # it's being cleaned up. Either way, this guarantees the "died" event is logged.
+    PODMAN_TIMEOUT=4 run_podman '?' wait $id
```

Comment:
Looking at this you could just rmeove the `-d` from podman run as the if podman run runs in the foreground it will ensure to delete the container before exit. I guess then we need another way to fetch the cid so I guess in terms of podman command nothing changed so the wait here is fine for me.